src/events/messageCreate/50_command-engine.js

```diff
@@ -1,5 +1,6 @@
 const commands = require('../../components/commands')
 const { commands: cmdAnalytics } = require('../../components/analytics')
+const i18n = require('../../internal/i18n')
```

Comment:
Ironically, this seems to contradict the contribution guideline that states that the `i18n` object is globally available and shouldn't be required on its own... Is there a reason for this to be here?

Reply:
VSCode auto-import 💯